ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/query/VaultNamedQuery.java

```diff
@@ -0,0 +1,43 @@
+package net.corda.v5.ledger.utxo.query;
+
+import net.corda.v5.base.annotations.DoNotImplement;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * An interface representing a named ledger query. A named ledger query must have a name associated with it,
+ * the rest of the fields are optional.
+ */
+@DoNotImplement
+public interface VaultNamedQuery {
```

Comment:
I don't think we need this interface. Do we?

Reply:
good point